acceptance_tests/features/steps/view_ce_ready_for_review_state.py

```diff
@@ -0,0 +1,33 @@
+from behave import given, when, then
+
+from acceptance_tests.features.pages import collection_exercise_details
+
+
+@given('the 201803 collection exercise for the QIFDI survey is Scheduled')
+def qifdi_201803_exists_and_scheduled_displayed(_):
+    collection_exercise_details.go_to('QBS', '1803')
```

Comment:
This function will need to check if the state is 'Scheduled' if not load events for this CE. (or you can pick a survey that already has events loaded in the setup)

Reply:
Do we have a Survey / CE with events that are already loaded (if not this one)?

Reply:
Any of the following should work although some may be getting used in other tests:
survey, period
023,1801
023,1802
023,1803
023,1804
023,1805
023,1806
023,1807
023,1808
023,1809
023,1810
023,1811
023,1812
073,201801
073,201802
134,201801
134,201802
134,201803
134,201804
134,201805
134,201806
134,201807
134,201808
134,201809
134,201810
134,201811
134,201812
139,1803
139,1806

Reply:
I used RSI 1809 and 1810 and it works fine